compiler/arser/include/arser/arser.h

```diff
@@ -119,10 +146,41 @@ enum class DataType
 
 class Arser;
 
+/**
+ * Argument
+ *   ├── positional argument
+ *   └── optioanl argument  [ dash at the beginning of the string ]
+ *       ├── long option    [ two or more dashes ]
```

Comment:
(Q) does `---` or `----` work?

Reply:
@seanshpark Yes. It works well. Which one will be long or short is intentionally not be described because that is so verbose.